src/Pages/BookDetails/BookDetails.js

```diff
@@ -1,2 +1,185 @@
-import React from "react";
-import ReactDOM from "react-dom";
+import React, { useState } from "react";
+import styled from "styled-components";
+import DescSection from "./DescSection";
+import FinishBook from "./FinishBook";
+import CommentSection from "./CommentSection";
+import IntroSection from "./IntroSection";
+import Aside from "./Aside";
+
+function BookDetails() {
+  // const [data, setData] = useState({ book: [] });
+  const [open, setOpen] = useState(false);
+
+  const handleMoreBtn = () => {
+    setOpen(!open);
+  };
+
+  // useEffect(() => {
+  //   const result = "data/data.json";
+  //   setData(result.data);
+  // }, []);
+
+  return (
+    <>
+      <Nav />
+      <Main>
+        <article>
+          <DescSection />
+          <BestSection>
+            <h1>에세이 분야 주간 베스트 도서</h1>
+          </BestSection>
+          <FinishBook />
+          <IntroSection />
+          <ContentSection open={open}>
+            <div>
+              <h1>목차</h1>
+              <img
+                onClick={handleMoreBtn}
```

Comment:
```suggestion
                onClick={() => setIsOpened(isOpened)}
```
기존 방식도 상관 없지만 함수형 컴포넌트 같은 경우 이런 식으로 작성하는 것이 가독성 측면에서 더 나아보입니다
- 만약 이름을 짓더라도 handle -> toggle로 바꾸는 것이 더 좋을 것 같네요

Reply:
네 !